distribution/windows/setup/prowlarr.iss

```diff
@@ -34,9 +34,10 @@ DisableReadyPage=True
 CompressionThreads=2
 Compression=lzma2/normal
 AppContact={#ForumsURL}
-VersionInfoVersion={#BaseVersion}.{#BuildNumber}
+VersionInfoVersion={#BaseVersion}
```

Comment:
BaseVersion here is actually the full version minus the build number. If you just want the actual major version (like Sonarr) this will need to be reworked a bit

https://github.com/Prowlarr/Prowlarr/blob/48cd1d9f6b55c71f42fab14871f14d7b15f4ed41/azure-pipelines.yml#L12

Reply:
fixup pushed; supposedly chatgpt thinks thatd work

Reply:
We'll see what the build artifact says

Reply:
chatgpt fixup! hopefully sorts it